lib/generators/suspenders/styles_generator.rb

```diff
@@ -3,9 +3,7 @@ module Generators
     class StylesGenerator < Rails::Generators::Base
       include Suspenders::Generators::APIAppUnsupported
 
-      CSS_OPTIONS = %w[tailwind postcss].freeze
-
-      class_option :css, enum: CSS_OPTIONS, default: "postcss"
+      class_option :css, enum: Generators::CSS_OPTIONS, default: "postcss"
```

Comment:
Since we don't provide a default for the `WebGenerator`, I wonder if we should remove it here? Or, is not having a default a smell?

My intent with not having a default is to force the consumer to pick. Not all projects have the same requirements, so I want this to be flexable.